.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: java
```

Comment:
You added many tools in this pull request. Would us not having these tools installed cause any build errors for us?